### ISSUE TITLE
Version 2.1.5 Doesn't load anymore

### ISSUE THREAD

**Issue by Gyztor** [New-Project-Final-Final-WIP/HeadlessTweaks#20]
Simply throws a no loadable mod error than nothing else. 2.1.4 still works as expected though.
```[ERROR][ResoniteModLoader] No loadable mod found in C:\Users\Admin\Documents\SteamCMD\steamapps\common\Resonite\Headless\rml_mods\HeadlessTweaks.dll```
![image](https://github.com/user-attachments/assets/160aef25-9d00-4de1-a3f9-2bff2e371b9b)

**Comment by badhaloninja** [comment]
Out of curiosity did you download the `0Harmony-Net8.dll` from the [ResoniteModLoader 3.0.0 release](https://github.com/resonite-modding-group/ResoniteModLoader/releases/tag/3.0.0) or the `0Harmony.dll` version?

**Comment by Gyztor** [comment]
The Net8 one. i re-downloaded it just to make sure nothing accidently got replaced and it still didn't launch.

**Comment by Gyztor** [comment]
I went looking and was wondering if it has something to do with the Target Framework not being net472. That is what i have seen other mods doing but not sure if that's it.

**Comment by XDelta** [comment]
Should also be closable. with latest release.

**Comment by Gyztor** [comment]
Yeah it appears to be working as of 2.1.9. Will go ahead and close this.
